modules/openapi/component-protocol/components/base/provider.go

```diff
@@ -70,11 +70,15 @@ func InitProvider(scenario, compName string) {
 
 // InitProviderWithCreator register component as provider with custom providerCreator.
 func InitProviderWithCreator(scenario, compName string, creator servicehub.Creator) {
+	InitProviderWithNameSpace(MakeComponentProviderName(scenario, compName), creator)
+}
+
+func InitProviderWithNameSpace(providerName string, creator servicehub.Creator) {
```

Comment:
Do not expose this method, rename to `initProviderWithCustomName`.
Add a new methods `InitProviderToDefaultNamespace` for default comp register.